src/cmake/openvino.cmake

```diff
@@ -20,8 +20,10 @@ endif()
 
 add_library(${TARGET_NAME}
     $<TARGET_OBJECTS:ngraph_obj>
+    $<TARGET_OBJECTS:ngraph_obj_version>
```

Comment:
do we need this explicitly? We already have `target_sources(${TARGET} PRIVATE $<TARGET_OBJECTS:${TARGET}_version>)` in `ov_add_version_defines`

Reply:
Yes but it is a private dependency openvino doesn't see these files.

Reply:
maybe we need to use `PUBLIC` in `target_sources` then?

Reply:
It doesn't work